tests/test_orm_1.py

```
@@ -0,0 +1,149 @@
+from cushy_storage.orm import QuerySet
+from tests.utils import delete_cache
+
+cache_file = {
+    "test_orm_add_and_query": "./cache/test-cushy-orm-cache",
+    "test_orm_delete": "./cache/test-cushy-orm-cache-orm-delete",
+    "test_orm_update": "./cache/test-cushy-orm-cache-orm-update",
+    "test_orm_set": "./cache/test-cushy-orm-cache-orm-set",
+    "test_orm_remove_duplicates": "./cache/test-cushy-orm-cache-orm-remove-duplicates",
+}
+
+def delete_cache():
```

Ruff (0.8.2)

Redefinition of unused `delete_cache` from line 8

(F811)